### PR TITLE
Changed machine-config-operator single-node aws e2e job for master-presubmit to required

### PR DESCRIPTION
**Pull request by omertuc**

### CODE REVIEW

**Comment by sinnykumari** [comment]
Can we first let it run for few days with always_run: true and then mark it as blocking? We are also near to feature freeze, if this job fails, it will cause delay in other PR merge.

**Comment by kikisdeliveryservice** [comment]
Agree 1000% with @sinnykumari 

/hold

File: ci-operator/jobs/openshift/machine-config-operator/openshift-machine-config-operator-master-presubmits.yaml [hunk 1]
```diff
@@ -402,7 +402,6 @@ presubmits:
       ci-operator.openshift.io/prowgen-controlled: "true"
       pj-rehearse.openshift.io/can-be-rehearsed: "true"
     name: pull-ci-openshift-machine-config-operator-master-e2e-aws-single-node
-    optional: true
```
Comment (kikisdeliveryservice) [inline ci-operator/jobs/openshift/machine-config-operator/openshift-machine-config-operator-master-presubmits.yaml:405]: undelete this

Comment (romfreiman) [reply ci-operator/jobs/openshift/machine-config-operator/openshift-machine-config-operator-master-presubmits.yaml:405]: @kikisdeliveryservice @sinnykumari feel free to adjust the PR. As far as you remember to set it a blocking afterwards, I dont have any objections

Comment (kikisdeliveryservice) [reply ci-operator/jobs/openshift/machine-config-operator/openshift-machine-config-operator-master-presubmits.yaml:405]: I'm fine waiting for omer to adjust it as we don't own this test.

**Review by kikisdeliveryservice — APPROVED** [review]
thanks @omertuc

**Comment by kikisdeliveryservice** [comment]
@sinnykumari PTAL and remove hold when you're done :)

/assign @sinnykumari

**Comment by kikisdeliveryservice** [comment]
not sure how that lgtm got on there :thinking: 

/lgtm cancel

**Comment by kikisdeliveryservice** [comment]
ugh how weird

/approve

**Pull request merged**